src/containers/shared/transactionUtils.ts

```diff
@@ -115,7 +115,7 @@ interface Tx {
   Flags: number
 }
 
-interface Transaction {
+export interface Transaction {
```

Comment:
This may be a more well-defined type to use in `TransactionDescriptionProps` instead of <T = any>